src/common/utils.ts

```diff
@@ -17,7 +17,7 @@ function dropsToXrp(drops: string | BigNumber): string {
   if (typeof drops === 'string') {
     if (!drops.match(/^-?[0-9]*\.?[0-9]*$/)) {
       throw new ValidationError(`dropsToXrp: invalid value '${drops}',` +
-        ` should be a number matching (^-?[0-9]*\.?[0-9]*$).`)
+        ` should be a number matching (^-?[0-9]*.?[0-9]*$).`)
```

Comment:
I know this is just the text of the error, but I think we want the `\` to be part of the regex here since `.` matches any character (but `\.` only matches `.`). In other words, `123.456` matches but `123a456` should not.

Reply:
that's a great point, however the linter caught that `'(^-?[0-9]*\.?[0-9]*$)'` was being shown to the user as `'(^-?[0-9]*.?[0-9]*$)'` already. I'll change this to `'(^-?[0-9]*\\.?[0-9]*$)'` instead to make sure that the backslash isn't escaped (linting ftw!)